modules/CONT-05_Merging-pull-requests.yml

```diff
@@ -46,7 +46,7 @@ screens:
       presenter-script:
         - Let's finish the workflow by merging our changes into master.
       steps:
-        - description: "Merge the pull request you created, closing your original issue with the merge commit."
+        - description: "Merge the pull request you created, closing your original issue with the merge commit. Don't forget Github looks for the 'Fixes' keyword."
```

Comment:
Do we specifically want to call out fixes? Or should we say something like, "Don't forget GitHub looks for the keywords: 'Fixes', 'Closes', or 'Resolves'."